docs/parser/flexer.md

```diff
@@ -88,6 +89,38 @@ they mostly choose the _longest_ match instead. Once the pattern is matched, the
 associated code is executed and the process starts over again until the input
 stream has been consumed.
 
+### Automated code generation
+In order to avoid getting lexer definition out of sync with its implementation
+(generated engine), it is required to create a separate crate for the generated 
+engine with lexer definition as one of its dependencies.
```

Comment:
```suggestion
In order to avoid the lexer definition getting out of sync with its implementation
(the generated engine), it is necessary to create a separate crate for the generated 
engine that has the lexer definition as one of its dependencies.
```